### ISSUE TITLE
[Modular CLI]: Cannot install on ubuntu version=22.04  codename=jammy arch=x86_64

### ISSUE THREAD

**Issue by skamensky** [modularml/mojo#626]
### Issue description

# Command

```
 curl https://get.modular.com | \
  MODULAR_AUTH=censored \
  sh -
```
# Environment info
`distro=ubuntu  version=22.04  codename=jammy  arch=x86_64`

# Similar reports
It does not seem the same as these two issues whose errors are related to `OpenPGP`.

https://github.com/modularml/mojo/issues/612
https://github.com/modularml/mojo/issues/606

# My Guess
My error seems to be related to a missing release repo for Ubuntu 22.04

# Log output

```
   OK: Checking for required executable 'curl' ...
   OK: Checking for required executable 'apt-get' ...
   OK: Detecting your OS distribution and release using system methods ...
 ^^^^: ... Detected/provided for your OS/distribution, version and architecture:
 >>>>:
 >>>>: ... distro=ubuntu  version=22.04  codename=jammy  arch=x86_64  
 >>>>:
 NOPE: Checking for apt dependency 'apt-transport-https' ...
 NOPE: Updating apt repository metadata cache ...
Failed to update via apt-get update
Get:1 https://download.docker.com/linux/ubuntu xenial InRelease [66.2 kB]
Hit:2 https://download.docker.com/linux/ubuntu jammy InRelease
Ign:3 http://dl.google.com/linux/chrome-remote-desktop/deb stable InRelease
Get:4 https://apt.releases.hashicorp.com jammy InRelease [12.9 kB]
Hit:5 https://linux.teamviewer.com/deb stable InRelease
Get:6 https://deb.nodesource.com/node_16.x jammy InRelease [4,583 B]
Hit:7 https://dl.google.com/linux/chrome/deb stable InRelease
Get:8 http://security.ubuntu.com/ubuntu jammy-security InRelease [110 kB]
Get:9 https://dl.yarnpkg.com/debian stable InRelease [17.1 kB]
Hit:10 http://dl.google.com/linux/chrome-remote-desktop/deb stable Release
Hit:11 http://packages.microsoft.com/repos/code stable InRelease
Hit:12 http://downloads.metasploit.com/data/releases/metasploit-framework/apt lucid InRelease
Err:1 https://download.docker.com/linux/ubuntu xenial InRelease
  The following signatures couldn't be verified because the public key is not available: NO_PUBKEY 7EA0A9C3F273FCD8
Hit:13 http://il.archive.ubuntu.com/ubuntu jammy InRelease
Hit:14 https://ppa.launchpadcontent.net/alessandro-strada/ppa/ubuntu jammy InRelease
Err:4 https://apt.releases.hashicorp.com jammy InRelease
  The following signatures couldn't be verified because the public key is not available: NO_PUBKEY AA16FCBCA621E701
Get:16 http://il.archive.ubuntu.com/ubuntu jammy-updates InRelease [119 kB]
Hit:15 https://hub-dist.unity3d.com/artifactory/hub-debian-prod-local stable InRelease
Hit:17 https://ppa.launchpadcontent.net/deadsnakes/ppa/ubuntu jammy InRelease
Get:18 https://pkgs.tailscale.com/stable/ubuntu kinetic InRelease
Err:9 https://dl.yarnpkg.com/debian stable InRelease
  The following signatures were invalid: EXPKEYSIG 23E7166788B63E1E Yarn Packaging <yarn@dan.cx>
Get:20 http://il.archive.ubuntu.com/ubuntu jammy-backports InRelease [109 kB]
Hit:21 https://ppa.launchpadcontent.net/graphics-drivers/ppa/ubuntu jammy InRelease
Ign:22 https://ppa.launchpadcontent.net/lzh9102/qwinff/ubuntu jammy InRelease
Get:23 https://repo.waydro.id jammy InRelease [1,309 B]
Hit:24 https://ppa.launchpadcontent.net/obsproject/obs-studio/ubuntu jammy InRelease
Get:25 http://security.ubuntu.com/ubuntu jammy-security/main amd64 DEP-11 Metadata [43.1 kB]
Get:26 http://security.ubuntu.com/ubuntu jammy-security/universe amd64 DEP-11 Metadata [40.1 kB]
Hit:27 https://ppa.launchpadcontent.net/openshot.developers/ppa/ubuntu jammy InRelease
Get:28 http://il.archive.ubuntu.com/ubuntu jammy-updates/main amd64 Packages [970 kB]
Hit:29 https://ppa.launchpadcontent.net/pipewire-debian/pipewire-upstream/ubuntu jammy InRelease
Err:23 https://repo.waydro.id jammy InRelease
  The following signatures couldn't be verified because the public key is not available: NO_PUBKEY 0E406D181DCEE19C
Hit:30 https://ppa.launchpadcontent.net/solaar-unifying/stable/ubuntu jammy InRelease
Get:31 http://il.archive.ubuntu.com/ubuntu jammy-updates/main i386 Packages [484 kB]
Get:32 http://il.archive.ubuntu.com/ubuntu jammy-updates/main amd64 DEP-11 Metadata [101 kB]
Get:33 http://il.archive.ubuntu.com/ubuntu jammy-updates/universe amd64 Packages [979 kB]
Get:34 http://il.archive.ubuntu.com/ubuntu jammy-updates/universe i386 Packages [653 kB]
Get:35 http://il.archive.ubuntu.com/ubuntu jammy-updates/universe amd64 DEP-11 Metadata [289 kB]
Hit:36 https://ppa.launchpadcontent.net/wereturtle/ppa/ubuntu jammy InRelease
Get:37 http://il.archive.ubuntu.com/ubuntu jammy-updates/multiverse amd64 DEP-11 Metadata [940 B]
Get:38 http://il.archive.ubuntu.com/ubuntu jammy-backports/main amd64 DEP-11 Metadata [4,928 B]
Get:39 http://il.archive.ubuntu.com/ubuntu jammy-backports/universe amd64 DEP-11 Metadata [16.6 kB]
Err:40 https://ppa.launchpadcontent.net/lzh9102/qwinff/ubuntu jammy Release
  404  Not Found [IP: 185.125.190.52 443]
Reading package lists...
W: GPG error: https://download.docker.com/linux/ubuntu xenial InRelease: The following signatures couldn't be verified because the public key is not available: NO_PUBKEY 7EA0A9C3F273FCD8
E: The repository 'https://download.docker.com/linux/ubuntu xenial InRelease' is not signed.
W: Target Packages (stable/binary-amd64/Packages) is configured multiple times in /etc/apt/sources.list.d/archive_uri-https_download_docker_com_linux_ubuntu-jammy.list:3 and /etc/apt/sources.list.d/docker.list:1
W: Target Packages (stable/binary-all/Packages) is configured multiple times in /etc/apt/sources.list.d/archive_uri-https_download_docker_com_linux_ubuntu-jammy.list:3 and /etc/apt/sources.list.d/docker.list:1
W: Target Translations (stable/i18n/Translation-en_IL) is configured multiple times in /etc/apt/sources.list.d/archive_uri-https_download_docker_com_linux_ubuntu-jammy.list:3 and /etc/apt/sources.list.d/docker.list:1
W: Target Translations (stable/i18n/Translation-en) is configured multiple times in /etc/apt/sources.list.d/archive_uri-https_download_docker_com_linux_ubuntu-jammy.list:3 and /etc/apt/sources.list.d/docker.list:1
W: Target Translations (stable/i18n/Translation-en_GB) is configured multiple times in /etc/apt/sources.list.d/archive_uri-https_download_docker_com_linux_ubuntu-jammy.list:3 and /etc/apt/sources.list.d/docker.list:1
W: Target DEP-11 (stable/dep11/Components-amd64.yml) is configured multiple times in /etc/apt/sources.list.d/archive_uri-https_download_docker_com_linux_ubuntu-jammy.list:3 and /etc/apt/sources.list.d/docker.list:1
W: Target DEP-11 (stable/dep11/Components-all.yml) is configured multiple times in /etc/apt/sources.list.d/archive_uri-https_download_docker_com_linux_ubuntu-jammy.list:3 and /etc/apt/sources.list.d/docker.list:1
W: Target DEP-11-icons-small (stable/dep11/icons-48x48.tar) is configured multiple times in /etc/apt/sources.list.d/archive_uri-https_download_docker_com_linux_ubuntu-jammy.list:3 and /etc/apt/sources.list.d/docker.list:1
W: Target DEP-11-icons (stable/dep11/icons-64x64.tar) is configured multiple times in /etc/apt/sources.list.d/archive_uri-https_download_docker_com_linux_ubuntu-jammy.list:3 and /etc/apt/sources.list.d/docker.list:1
W: Target DEP-11-icons-hidpi (stable/dep11/icons-64x64@2.tar) is configured multiple times in /etc/apt/sources.list.d/archive_uri-https_download_docker_com_linux_ubuntu-jammy.list:3 and /etc/apt/sources.list.d/docker.list:1
W: Target CNF (stable/cnf/Commands-amd64) is configured multiple times in /etc/apt/sources.list.d/archive_uri-https_download_docker_com_linux_ubuntu-jammy.list:3 and /etc/apt/sources.list.d/docker.list:1
W: Target CNF (stable/cnf/Commands-all) is configured multiple times in /etc/apt/sources.list.d/archive_uri-https_download_docker_com_linux_ubuntu-jammy.list:3 and /etc/apt/sources.list.d/docker.list:1
W: An error occurred during the signature verification. The repository is not updated and the previous index files will be used. GPG error: https://apt.releases.hashicorp.com jammy InRelease: The following signatures couldn't be verified because the public key is not available: NO_PUBKEY AA16FCBCA621E701
W: https://dl.yarnpkg.com/debian/dists/stable/InRelease: Key is stored in legacy trusted.gpg keyring (/etc/apt/trusted.gpg), see the DEPRECATION section in apt-key(8) for details.
W: An error occurred during the signature verification. The repository is not updated and the previous index files will be used. GPG error: https://dl.yarnpkg.com/debian stable InRelease: The following signatures were invalid: EXPKEYSIG 23E7166788B63E1E Yarn Packaging <yarn@dan.cx>
W: http://downloads.metasploit.com/data/releases/metasploit-framework/apt/dists/lucid/InRelease: Key is stored in legacy trusted.gpg keyring (/etc/apt/trusted.gpg), see the DEPRECATION section in apt-key(8) for details.
W: An error occurred during the signature verification. The repository is not updated and the previous index files will be used. GPG error: https://repo.waydro.id jammy InRelease: The following signatures couldn't be verified because the public key is not available: NO_PUBKEY 0E406D181DCEE19C
E: The repository 'https://ppa.launchpadcontent.net/lzh9102/qwinff/ubuntu jammy Release' does not have a Release file.
W: Target Packages (stable/binary-amd64/Packages) is configured multiple times in /etc/apt/sources.list.d/archive_uri-https_download_docker_com_linux_ubuntu-jammy.list:3 and /etc/apt/sources.list.d/docker.list:1
W: Target Packages (stable/binary-all/Packages) is configured multiple times in /etc/apt/sources.list.d/archive_uri-https_download_docker_com_linux_ubuntu-jammy.list:3 and /etc/apt/sources.list.d/docker.list:1
W: Target Translations (stable/i18n/Translation-en_IL) is configured multiple times in /etc/apt/sources.list.d/archive_uri-https_download_docker_com_linux_ubuntu-jammy.list:3 and /etc/apt/sources.list.d/docker.list:1
W: Target Translations (stable/i18n/Translation-en) is configured multiple times in /etc/apt/sources.list.d/archive_uri-https_download_docker_com_linux_ubuntu-jammy.list:3 and /etc/apt/sources.list.d/docker.list:1
W: Target Translations (stable/i18n/Translation-en_GB) is configured multiple times in /etc/apt/sources.list.d/archive_uri-https_download_docker_com_linux_ubuntu-jammy.list:3 and /etc/apt/sources.list.d/docker.list:1
W: Target DEP-11 (stable/dep11/Components-amd64.yml) is configured multiple times in /etc/apt/sources.list.d/archive_uri-https_download_docker_com_linux_ubuntu-jammy.list:3 and /etc/apt/sources.list.d/docker.list:1
W: Target DEP-11 (stable/dep11/Components-all.yml) is configured multiple times in /etc/apt/sources.list.d/archive_uri-https_download_docker_com_linux_ubuntu-jammy.list:3 and /etc/apt/sources.list.d/docker.list:1
W: Target DEP-11-icons-small (stable/dep11/icons-48x48.tar) is configured multiple times in /etc/apt/sources.list.d/archive_uri-https_download_docker_com_linux_ubuntu-jammy.list:3 and /etc/apt/sources.list.d/docker.list:1
W: Target DEP-11-icons (stable/dep11/icons-64x64.tar) is configured multiple times in /etc/apt/sources.list.d/archive_uri-https_download_docker_com_linux_ubuntu-jammy.list:3 and /etc/apt/sources.list.d/docker.list:1
W: Target DEP-11-icons-hidpi (stable/dep11/icons-64x64@2.tar) is configured multiple times in /etc/apt/sources.list.d/archive_uri-https_download_docker_com_linux_ubuntu-jammy.list:3 and /etc/apt/sources.list.d/docker.list:1
W: Target CNF (stable/cnf/Commands-amd64) is configured multiple times in /etc/apt/sources.list.d/archive_uri-https_download_docker_com_linux_ubuntu-jammy.list:3 and /etc/apt/sources.list.d/docker.list:1
W: Target CNF (stable/cnf/Commands-all) is configured multiple times in /etc/apt/sources.list.d/archive_uri-https_download_docker_com_linux_ubuntu-jammy.list:3 and /etc/apt/sources.list.d/docker.list:1
 ^^^^: ... Failed to update via apt-get update - Context above.
 !!!!: Oh no, your setup failed! :-( ... But we might be able to help. :-)
 !!!!: 
 !!!!: You can contact Modular for further assistance.
 !!!!: 
```

### Steps to reproduce

- Include relevant code snippet that did not work as expected.
- If applicable, add screenshots to help explain the problem.
- Include anything else that might help us debug the issue.

### Version Info

```shell
- Provide Modular CLI version by pasting the output of `modular -v`
- What OS did you install modular CLI on ?
```

**Comment by codingfun2022** [comment]
```text
W: GPG error: https://download.docker.com/linux/ubuntu xenial InRelease: The following signatures couldn't be verified because the public key is not available: NO_PUBKEY 7EA0A9C3F273FCD8
```
Might be the same as #606 , try enabling VPN or other network proxies.

**Comment by skamensky** [comment]
I had a general problem with my package repositories. After cleaning them up, modular installed successfully!
